### PR TITLE
Delete rospack find from CMakeLists.txt and use the result of find package

### PR DESCRIPTION
**Pull request by aginika**

### CODE REVIEW

**Comment by garaemon** [comment]
I think it does not work if you install baxter-description from deb file.
Please check https://github.com/jsk-ros-pkg/jsk_demos/blob/master/jsk_maps/CMakeLists.txt#L29
You may need to check SOURCE_DIR, SOURCE_PREFIX and PREFIX

**Comment by aginika** [comment]
Thank you!
I changed to write like jsk_maps/CMakeLists.txt.

File: jsk_baxter_robot/jsk_baxter_web/CMakeLists.txt [hunk 1]
```diff
@@ -8,8 +8,18 @@ find_package(catkin REQUIRED COMPONENTS
   tf2_web_republisher
 )
 
-execute_process(COMMAND rospack find baxter_description OUTPUT_VARIABLE baxter_description_PACKAGE_PATH OUTPUT_STRIP_TRAILING_WHITESPACE)
-execute_process(COMMAND cp -Ru ${baxter_description_PACKAGE_PATH}  ${PROJECT_SOURCE_DIR}/www/ OUTPUT_VARIABLE OUTPUT OUTPUT_STRIP_TRAILING_WHITESPACE)
+if (EXISTS ${baxter_description_SOURCE_DIR}/meshes)
+  set(_baxter_description_location ${baxter_description_SOURCE_DIR}/meshes)
+elseif (EXISTS ${baxter_description_SOURCE_PREFIX}/meshes)
+  set(_baxter_description_location ${baxter_description_SOURCE_PREFIX}/meshes)
+elseif (EXISTS ${baxter_description_PREFIX}/meshes)
```
Comment (garaemon) [inline jsk_baxter_robot/jsk_baxter_web/CMakeLists.txt]: i think this should be `${baxter_description_PREFIX}/share/baxter_description`

**Comment by aginika** [comment]
Thank you. I updated again.

**Comment by garaemon** [comment]
LGTM

**Pull request merged**